src/app/view/components/dot-workflow-task-detail/services/dot-workflow-task-detail.service.ts

```diff
@@ -16,8 +16,8 @@ export interface DotTaskAction {
  */
 @Injectable()
 export class DotWorkflowTaskDetailService {
-    private data: Subject<DotTaskAction> = new Subject();
-    private _header: Subject<string> = new Subject();
+    private data: BehaviorSubject<DotTaskAction> = new BehaviorSubject(null);
```

Comment:
This is the fix, there was a race condition so when we subscribe to the `viewUrl$` the Observable was already sent, by using `BehaviorSubject` we prevent that because it will always sent you the current value on subscription.